src/directives/decorators/bootstrap/default.html

```diff
@@ -1,20 +1,27 @@
 <div class="form-group schema-form-{{form.type}} {{form.htmlClass}}"
      ng-class="{'has-error': hasError(), 'has-success': hasSuccess(), 'has-feedback': form.feedback !== false }">
   <label class="control-label" ng-class="{'sr-only': !showTitle()}" for="{{form.key.slice(-1)[0]}}">{{form.title}}</label>
-
-  <input ng-show="form.key"
-         type="{{form.type}}"
-         step="any"
-         sf-changed="form"
-         placeholder="{{form.placeholder}}"
-         class="form-control {{form.fieldHtmlClass}}"
-         id="{{form.key.slice(-1)[0]}}"
-         ng-model-options="form.ngModelOptions"
-         ng-model="$$value$$"
-         ng-disabled="form.readonly"
-         schema-validate="form"
-         name="{{form.key.slice(-1)[0]}}"
-         aria-describedby="{{form.key.slice(-1)[0] + 'Status'}}">
+  <div ng-class="{'input-group': (form.fieldAddonLeft || form.fieldAddonRight)}">
```

Comment:
I like your commit in general, but this introduces an extra div for everyone... I'm not sure its a good idea, but maybe we could try having two versions and switching between them with an ng-if?

Something like:
```
<div class="input-group" ng-if="(form.fieldAddonLeft || form.fieldAddonRight)}">
...
</div>
<input  ng-if="!(form.fieldAddonLeft || form.fieldAddonRight)}"  .... >
```
What do you think?